interface/super/edit_list.php

```diff
@@ -818,147 +810,162 @@ function mysubmit() {
 </head>
 
 <body class="body_top">
+<nav class="navbar navbar-default navbar-fixed-top">
+<div class="container-fluid">
```

Comment:
Container-fluid is required in this instance because of the `navbar-fixed-top` - basically if I want to ensure full width usage I go with the fluid container, otherwise I stick with a regular container. But for the navbar, I definitely want it full width, hence the `fluid`